test/integration/patients/patient/sidebar.js

```diff
@@ -386,6 +434,76 @@ context('patient sidebar', function() {
       .get('.modal')
       .should('not.exist');
 
+    cy
+      .routeForm(fx => {
+        fx.data = testReadOnlyForm;
+
+        return fx;
+      });
```

Comment:
This doesn't work and causes the test to fail.

The first `cy.routeForm()` in this test loads a script reducer form. And then here I want to override that and route the `testReadOnlyForm`.

We can do that with the other intercepts like `routeAction`, `routeFlow`, etc, but it doesn't seem to work for `routeForm`. So still trying to figure that out.